api/v1beta1/hyperconverged_types.go

```diff
@@ -244,6 +249,11 @@ type HyperConvergedFeatureGates struct {
 	// +optional
 	// +kubebuilder:default=true
 	EnableCommonBootImageImport bool `json:"enableCommonBootImageImport"`
+
+	// deploy Tekton tasks operator, which deploys kubevirt tekton tasks and example pipelines
+	// +optional
+	// +kubebuilder:default=false
+	EnableTektonTaskOperator bool `json:"enableTektonTaskOperator"`
```

Comment:
Seems not in use.

Also, HCO can't control the operator itself. Do you min the TTO CR? If yes, please consider renaming this FG.